pandas/tests/reshape/test_concat.py

```diff
@@ -2487,3 +2487,14 @@ def test_concat_aligned_sort_does_not_raise():
                             columns=[1, 'a'])
     result = pd.concat([df, df], ignore_index=True, sort=True)
     tm.assert_frame_equal(result, expected)
+
+
+@pytest.mark.parametrize("s1name, s2name", [(np.int64(190), (43, 0)),
+                                            (190, (43, 0))])
```

Comment:
Could probably push the tuple on the above line down here - would make the indentation more readable